graylog2-web-interface/src/views/components/sidebar/NavItem.jsx

```diff
@@ -13,80 +13,106 @@ type Props = {
   icon: string,
   onClick: () => void,
   showTitleOnHover: boolean,
+  sidebarIsPinned: boolean,
+};
+
+type ContainerProps = {
+  isSelected: boolean,
+  showTitleOnHover: boolean,
+  sidebarIsPinned: boolean,
 };
 
 const Title: StyledComponent<{}, ThemeInterface, HTMLDivElement> = styled.div(({ theme }) => css`
   display: none;
   position: absolute;
-  padding: 5px 10px;
+  padding: 0 10px;
   left: 100%;
-  font-size: ${theme.fonts.size.body};
-  color: white;
-  background-color: ${theme.utils.contrastingColor(theme.colors.gray[10], 'AA')};
+  top: calc(50% - 13px);
+  height: 26px;
+  background-color: ${theme.colors.variant.lighter.info};
```

Comment:
The blue looks a little too dark in my opinion.
![image](https://user-images.githubusercontent.com/46300478/88904102-ed631180-d254-11ea-8e36-ef6affc3f832.png)

While the next lighter blue would look a little too bright. What do you think about using the same styles like for an active pagination list item:

![image](https://user-images.githubusercontent.com/46300478/88903740-6dd54280-d254-11ea-9a72-9713d665188c.png)

I used

```
  background-color: ${theme.colors.variant.lightest.info};
  border-top: 1px solid ${theme.colors.variant.lighter.info};
  border-right: 1px solid ${theme.colors.variant.lighter.info};
  border-bottom: 1px solid ${theme.colors.variant.lighter.info};
```

for the title + 
```
 background-color: ${isSelected ? theme.colors.variant.lighter.default : theme.colors.variant.lightest.info};
 border-bottom: 1px solid ${theme.colors.variant.lighter.info};
 border-left: 1px solid ${theme.colors.variant.lighter.info};
```
for Container -> IconWrapper -> ::after

Reply:
**regular**
![image](https://user-images.githubusercontent.com/122591/88949844-682e2b80-d259-11ea-98b1-309a236a1d0f.png)

**pinned**
![image](https://user-images.githubusercontent.com/122591/88949863-72502a00-d259-11ea-9a90-423e8d958a95.png)

Reply:
I love it! Thanks for the effort.